src/Fixer/Alias/NoAliasFunctionsFixer.php

```diff
@@ -107,6 +180,17 @@ public function configure(array $configuration = null)
                 $this->aliases = array_merge($this->aliases, self::$mbregSet);
                 $this->aliases = array_merge($this->aliases, self::$timeSet);
                 $this->aliases = array_merge($this->aliases, self::$exifSet);
+                $this->aliases = array_merge($this->aliases, self::$snmpSet);
```

Comment:
please sort in alphabetical order, also the if statements from line 198 till 229